fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -1126,6 +1126,28 @@ class BrowserRobot {
         assertItemWithResIdExists(itemWithResId("errorTryAgain"))
     }
 
+    fun verifyAddressNotFoundErrorMessage() {
+        assertItemContainingTextExists(
+            itemContainingText(getStringResource(R.string.mozac_browser_errorpages_unknown_host_title)),
+            itemContainingText("The browser could not find the host server for the provided address."),
+            itemContainingText("Check the address for typing errors such as "),
+            itemContainingText("ww"),
+            itemContainingText(".example.com instead of "),
+            itemContainingText("www"),
+            itemContainingText(".example.com."),
+            itemContainingText("If you are unable to load any pages, check your device’s data or Wi-Fi connection."),
+        )
+        assertItemWithResIdExists(itemWithResId("errorTryAgain"))
+    }
```

Comment:
I really don't know how stringent we need to be to checking for string precision here, this feels a bit excessive. I think it would be fine if we just check for the presence of a view ID or header to confirm that in view. What do you think @sv-ohorvath @AndiAJ

Reply:
Sounds good @AaronMT